src/main/java/com/gamingmesh/jobs/listeners/JobsListener.java

```diff
@@ -715,4 +716,9 @@ public void PlayerItemBreakEvent(PlayerItemBreakEvent event) {
     public void PlayerItemBreakEvent(InventoryClickEvent event) {
 	Jobs.getPlayerManager().resetiItemBonusCache(((Player) event.getWhoClicked()).getUniqueId());
     }
+
+	@EventHandler
+	public void PlayerSwapHandItemsEvent(PlayerSwapHandItemsEvent event) {

```

Comment:
Please try to follow method naming conventions. `lowerCamelCase`

Reply:
Yeah, I originally followed the style of the method above the one this PR adds. Changed to `lowerCamelCase` in 00289a6.